soc/xtensa/intel_adsp/cavs_v25/linker.ld

```diff
@@ -476,22 +490,25 @@ SECTIONS
     *(.gna_model)
     _data_end = ABSOLUTE(.);
     . = ALIGN(4096);
-  } >ram :ram_phdr
-  .lit4 : ALIGN(4)
+  } >ucram :ucram_phdr
+
+  .lit4 SEGSTART_CACHED : ALIGN(4)
   {
     _lit4_start = ABSOLUTE(.);
     *(*.lit4)
     *(.lit4.*)
     *(.gnu.linkonce.lit4.*)
     _lit4_end = ABSOLUTE(.);
   } >ram :ram_phdr
-  .cached :
-  {
-    *(.cached .cached.*)
-  } >ram :ram_phdr
+
+#undef RAMABLE_REGION
+#undef ROMABLE_REGION
+#define RAMABLE_REGION ucram :ucram_phdr
+#define ROMABLE_REGION ucram :ucram_phdr
+
 #include <linker/common-ram.ld>
 
-  .AltBootManifest : ALIGN(8)
+  .AltBootManifest SEGSTART_CACHED : ALIGN(8)
```

Comment:
a.8 doesn't have this... Does this have to be cached or uncached? Can try to keep this cached for now, it doesn't seem to contain any volatile data

Reply:
I tried both and didn't make a difference to this problem.